api/HTMLElement.json

```
@@ -420,6 +420,7 @@
       },
       "autocapitalize": {
         "__compat": {
+          "spec_url": "https://html.spec.whatwg.org/multipage/interaction.html#dom-autocapitalize",
```

There's a "For web developers" version of this one:

```suggestion
          "spec_url": "https://html.spec.whatwg.org/multipage/interaction.html#dom-autocapitalize-dev",
```

I haven't checked all of them, but I suspect there are many more cases like this. I would suggest trying to append "-dev" to all URLs and checking if that URL is valid, and if so use that instead.

cc @sideshowbarker for other tips.